packages/tools/src/types/EventTypes.ts

```diff
@@ -51,6 +51,8 @@ type AnnotationModifiedEventDetail = {
 type AnnotationRemovedEventDetail = {
   /** The annotation that is being added to the annotations manager. */
   annotation: Annotation;
+  /** FrameOfReferenceUID */
+  FrameOfReferenceUID: string;
```

Comment:
What's needed is the _managers_ UID, not the FrameOfReferenceUID, right?

Reply:
Oh you are using the correct value, but the name is wrong, its not a FrameOfReferenceUID, its an `FrameOfReferenceSpecificAnnotationManagerUID`..., which I understand is very long. Maybe `annotationManagerUID`?

Reply:
oh right, my bad